osmosis/assetlist.json

```diff
@@ -1105,6 +1107,7 @@
           }
         }
       ],
+      "coingecko_id": "pstake-staked-osmo",
```

Comment:
Please remove this cgid (we only want them defined in the assetlist corresponding to the (usually just one) chain they are originally issued).

Reply:
fixed